processing-service-core/api/models/db/tables/tenantConfig.js

```diff
@@ -43,6 +43,6 @@ var tableDefinition = {
 
 module.exports = {
     definition: tableDefinition,
-    name: "globalConfig",
+    name: "tenantConfig",
```

Comment:
Is this change actually related to this Jira?  If not, then remove this to keep this PR focused.  Create a separate branch and PR for unrelated issues that you come across.  They don't necessarily need a Jira if they are small change (although they might if they need independent QA).